recipes/lmfit/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.8.3" %}
+
+package:
+  name: lmfit
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/lmfit/lmfit-py.git
+  git_tag : {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
lmfit does not use numpy's C api. These are included in the build requirements because the setup.py for lmfit lists numpy and scipy in the install_requires.  As far as I understand, anything that is listed in the install_requires section of the setup.py needs to go in the build requirements or conda build will complain about setuptools downloading being disabled. Is that still true?